src/definitions/bluebps.json

```diff
@@ -0,0 +1,254 @@
+                        "type": "u16",
+                        "description": "The main battery voltage",
+                        "units": "millivolt"
+                    },
+                    {
+                        "name": "battery_current",
+                        "type": "u16",
+                        "description": "The current measurement",
+                        "units": "centiAmp"
+                    },
+                    {
+                        "name": "battery_temperature",
+                        "type": "u16",
+                        "description": "The battery temperature",
+                        "units": "decidegrees Centigrade"
+                    },
+                    {
+                        "name": "cpu_temperature",
+                        "type": "u16",
+                        "description": "The cpu temperature",
+                        "units": "millisecond"
+                    },
+                    {
+                        "name": "flags",
+                        "type": "u8",
+                        "description": "flags indicating if any of the configured limits are currently exceeded"
+                    },
+                    {
+                        "name": "cell_voltages",
+                        "type": "vector",
+                        "vector": {
+                            "sizetype": "u8",
+                            "datatype": "u16",
+                            "size": "dynamic"
+                        },
+                        "description": "Array containing cell voltages",
+                        "units": "millisecond"
+                    }
+                ]
+            },
+            "events": {
+                "id": "9107",
+                "description": "A record of events causing a power lock-out. These numbers are non-volatile and reset only with the erase_flash control message.",
+                "payload": [
+                    {
+                        "name": "voltage",
+                        "type": "u16",
+                        "description": "The number of under-voltage events"
+                    },
+                    {
+                        "name": "current",
+                        "type": "u16",
+                        "description": "The number of over-current events"
+                    },
+                    {
+                        "name": "temperature",
+                        "type": "u16",
+                        "description": "The number of over-temperature events"
+                    }
+                ]
+            }
+        },
+        "control": {
+            "erase_flash": {
+                "id": "1100",
+                "description": "Erase flash, including parameter configuration and event counters. The mcu has a limited number of write/erase cycles (1k)!",
+                "payload": []
+            },
+            "reset_defaults": {
+                "id": "1400",
```

Comment:
This id is used by Ping1D please avoid 1400 space.

Reply:
They are (tentatively) allowed to collide, but I agree it should be adjusted in case we have problems with that allowance.

I will change it

Reply:
If the sensor is in the same bus, it'll be a problem, there is plenty of space to avoid this problem, there is no necessity to use the same message space as ping.